e2e/testcases/stress_test.go

```diff
@@ -332,11 +307,89 @@ func TestStress100CRDs(t *testing.T) {
 	}
 }
 
-// TestStressMemoryUsage applies 100 CRDs and then 10 objects for each resource.
-// This stressed both the number of watches and the number of objects,
+// TestStressManyDeployments applies 1000 Deployments with 1 replica each.
+// This ensures that Config Sync can apply resources while the cluster control
```

Comment:
100 or 1000?

Reply:
oops. I set it to 100 for a quick test and forgot to set it back. I'm running 1000 right now locall to check that it still works.

Reply:
fixed to 1000.